core/tests/protractor_utils/editor.js

```diff
@@ -139,9 +139,25 @@ var expectContentTextToEqual = function(text) {
 
 // INTERACTIONS
 
+// Used as the standard way to specify interaction for most purposes.
```

Comment:
This function should be used as the standard way to specify interactions for most purposes.